### PR TITLE
feat: Move SDK perfromance docs to platforms

### PR DESCRIPTION
**Pull request by HazAT**
This basically moves the performance Getting started into SDK platforms.

Supersedes:

https://github.com/getsentry/sentry-docs/pull/2121
https://github.com/getsentry/sentry-docs/pull/2133

### CODE REVIEW

**Review by dcramer — APPROVED** [review]
nice!

File: src/platforms/javascript/common/performance.mdx [hunk 1]
```diff
@@ -1,91 +1,58 @@
...
 ```
 
-Alternatively, instead of npm packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/tracing`:
-
-<JsCdnTag tracing />
-
-Next, initialize the integration in your call to `Sentry.init`:
-
-```js
+```javascript {tabTitle: CDN}
```
Comment (PeloWriter) [inline src/platforms/javascript/common/performance.mdx:28]: In this PR (https://github.com/getsentry/sentry-docs/pull/2160) , the CDN pieces moved into their own page (https://docs.sentry.io/platforms/javascript/configuration/install-cdn) based on feedback on the JS Getting Started content.

File: src/platforms/javascript/common/performance.mdx [hunk 2]
```diff
@@ -296,3 +288,8 @@ span.setData("parameters", parameters);
 span.setData("user_id", parameters.user_id);
 http.get(`${base_url}/${endpoint}/`, (data = parameters));
 ```
+
```
Comment (PeloWriter) [inline src/platforms/javascript/common/performance.mdx:291]: We've removed the Next Steps links that are hand managed in the SDK content - the goal will be to replace them with something that's auto-generated.

File: src/platforms/node/guides/koa/index.mdx [hunk 3]
```diff
@@ -40,3 +40,122 @@ app.on("error", (err, ctx) => {
...
+    scope.addEventProcessor(event => {
+      return Sentry.Handlers.parseRequest(event, ctx.request);
+    });
+    Sentry.captureException(err);
+  });
+});
+// the rest of your app
+```
+
+**Subsequent manual child spans**
```
Comment (PeloWriter) [inline src/platforms/node/guides/koa/index.mdx:139]:
```suggestion
### Subsequent Manual Child Spans
```

**Review by PeloWriter — APPROVED** [review]
Thank you!

**Pull request merged**